### PR TITLE
feat(init-generator): configure workbox-webpack-plugin as opted

### PR DESCRIPTION
**Pull request by jamesgeorge007**
**What kind of change does this PR introduce?**
Feature

**Did you add tests for your changes?**
Yes

**If relevant, did you update the documentation?**
Yes

**Summary**
#2430
Added a prompt for [workbox-webpack-plugin](https://developers.google.com/web/tools/workbox/modules/workbox-webpack-plugin) as part of `init-generator`.

**Does this PR introduce a breaking change?**
No

**Other information**
N/A

### CODE REVIEW

**Review by rishabh3112 — CHANGES_REQUESTED** [review]
Left few recommendations.

File: packages/generators/init-template/default/webpack.configjs.tpl [hunk 1]
```diff
@@ -28,6 +29,8 @@ const config = {
         new HtmlWebpackPlugin({
             template: 'index.html',
         }),
+<% } %><% if (workboxWebpackPlugin) { %>
```
[the remark this answers is not in the document]
Comment (evenstensberg) [reply packages/generators/init-template/default/webpack.configjs.tpl]: why? It's good for development too

**Review by alexander-akait — COMMENTED** [review]
Let's fix tests, I think for ideal testing we should run webpack after create configuration so we will ensure all works fine, but let's it improve in future PRs

**Review by evenstensberg — APPROVED** [review]
lgtm, even though PWA can be in dev mode too

**Review by alexander-akait — APPROVED** [review]

**Comment by alexander-akait** [comment]
Thanks

**Pull request merged**